tests/storage/helper.py

```diff
@@ -1,23 +1,55 @@
 import os
 import glob
+from test_mount_sanity import MountSanity
+from test_mount import Mount
 from test_throughput import Throughput
 from test_checksum import Checksum
+from test_write import Write
+from test_exists import Exists
+from test_delete import Delete
+
+
 
 
 def run_storage(tasks):
```

Comment:
Since I haven't been discussing code with you - and you probably agreed something else with Enric -, consider this comment as a future improvement and not something to be done now.

This approach is not modular and you repeat a lot of code here. For example: if I don't wan't to test something, I would not provide that test in the yaml file. But then your code would break because it's waiting for some specific variable.

How I would improve:
You have a yaml file that describes the tests we want to run, with a specific name given (which corresponds to the test class name). You also have a clear structure (interface) that the tests themselves should implement (ok, the constructor parameters are not the same but very similar, etc). So, I would just read the yaml file, iterate through the tests names, and dynamically try to import a class with that name and get the output.

Something like (pseudo code):
```
for test_name in tasks.keys:
    from test_name import test_name.upper_case() as test
    testing = test(tasks[test_name])
    testing.exit_code()
```

Reply:
This way it would be easy to run just the tests we want and to even add some new tests (which would be just creating the py file with the corresponding class and nothing more).

Reply:
And to avoid duplication of the filename, for example, you could set a "common_configs" entry in your yaml file, that would be merged with the configs of each test.

Reply:
Yes, you are right! I would implement this in separate PR.